cli-tool/visio_temp_file_remover.py

```
@@ -236,43 +79,25 @@ def load_config():
-            continue
-    
-    # If we're on Windows, PowerShell should always be available
-    # So as a last resort, if we're on Windows, we'll assume PowerShell is available
-    if platform.system() == 'Windows':
-        print_status("PowerShell detection failed, but since we're on Windows, assuming it's available.", "warning")
-        return True
-        
-    return False
+    try:
+        result = subprocess.run(
+            ["powershell", "-Command", "Write-Output 'PowerShell Test'"],
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)